cli/cmd/describe.go

```diff
@@ -20,6 +20,32 @@ var describeCmd = &cobra.Command{
 	Use:   "describe",
 	Short: "Show details of a specific odigos entity",
 	Long:  `Print detailed description of a specific odigos entity, which can be used to troubleshoot issues`,
```

Comment:
Do we need to update these descriptions? or have different ones for `odigos describe` and `odigos describe source`?